include/llfio/v2.0/path_view.hpp

```diff
@@ -1568,21 +1568,19 @@ class LLFIO_DECL path_view_component
   LLFIO_TREQUIRES(LLFIO_TPRED(is_source_acceptable<T>),
```

Comment:
`render()`, immediately above this, was removed in P1036R6. Consistency dictates that either it should go away or also remove the first argument.

Reply:
Indeed, I missed the history entries
* R5: `render()` function added as per LEWG request
* R6: `render()` function removed as per LEWG request

🙃

However, given that `render()` has existed for almost two years, I would first deprecate it for one "release" and then remove it. That should ease the transition for our user base considerably at a very low cost.

Reply:
I think the number of users of those functions is likely very low, as they were only added by LEWG and as you correctly point out, were clearly added wrong by me likely very late at night when my eyes weren't working.

P1030R7 is nearing completion after a month of working on it, so I'm going to go ahead and merge this. Thanks for the changeset.